### DISCUSSION TITLE
Set Batch Commands Property Double - no longer exists?

### DISCUSSION THREAD

**Discussion by bakalarczuk** [JSBSim-Team/jsbsim#1018] [Support requests]
In use pin  <Unnamed>  no longer exists on node  Set Batch Commands Property Double . Please refresh node or break links to remove pin.

How to update it to newest JSBSim?

**Comment by gallonmate** [comment]
Are you refereeing to the UE5 plugin? Can you show a screenshot of the issue and node? 
I'm not aware of any current issues like this.

**Comment by bakalarczuk** [comment]
Yes, I'm in UE 5.3.2, using jsbsim version 1.2.0
Here is screen of problematic part of blueprint with error log
<img width="965" alt="image" src="https://github.com/JSBSim-Team/jsbsim/assets/32766530/51487b4c-ae87-4564-a1de-342cc75e8c30">

**Comment by gallonmate** [comment]
That appears to be code for a custom/modified version of the plugin. Those nodes don't exist in the default plugin. Make sure you are using the custom plugin and not the default plugin from this repository. 

The blueprint error is stating that either there is an issue with your custom plugin not having been compiled or you are using the default plugin from this repository which doesn't have those custom nodes. 

The default plugin only has these command nodes as shown here:
![image](https://github.com/JSBSim-Team/jsbsim/assets/28702753/8bc716dc-d4a0-43e8-a5c0-8dca4046ee66)


Here's some example of usage of the default command nodes: https://github.com/JSBSim-Team/jsbsim/tree/master/UnrealEngine#extended-commands-and-properties

**Reply by Marek-Bakalarczuk** [reply]
Thanks.

**Comment by 140695809** [comment]
Do you understand the meaning of ”Set Batch Commands Property Double”？
